statsmodels/genmod/generalized_linear_model.py

```diff
@@ -1084,31 +1098,33 @@ def __init__(self, model, params, normalized_cov_params, scale,
 
     @cache_readonly
     def resid_response(self):
-        return self._data_weights * (self._endog-self.mu)
+        return self._freq_weights * (self._endog-self.mu)
 
     @cache_readonly
     def resid_pearson(self):
-        return (np.sqrt(self._data_weights) * (self._endog-self.mu) /
+        return (np.sqrt(self._freq_weights) * (self._endog-self.mu) /
                 np.sqrt(self.family.variance(self.mu)))
 
     @cache_readonly
     def resid_working(self):
         val = (self.resid_response / self.family.link.deriv(self.mu))
-        val *= self._data_weights
+        val *= self._freq_weights
         return val
 
     @cache_readonly
     def resid_anscombe(self):
+        # TODO: Data weights?
```

Comment:
This depends a bit on how we **define** the residuals, per unit/observation or with weights
In linear model WLS, we distinguish between `resid` and `wresid`